src/main/java/com/foodybuddy/dao/OrderDAO.java

```diff
@@ -4,12 +4,47 @@
 
 import com.foodybuddy.model.Order;
 
+/**
+ * The Interface OrderDAO.
+ */
 public interface OrderDAO {
-	
-	public void insert(Order c);
+
+	/**
+	 * Insert.
+	 *
+	 * @param c the c
+	 * @return 
+	 */
+	public Order insert(Order c);
+
+	/**
+	 * Gets the list by buyer id.
+	 *
+	 * @param buyerId the buyer id
+	 * @return the list by buyer id
+	 */
 	public List<Order> getListByBuyerId(int buyerId);
```

Comment:
@kanmaytacker  I was not allowing NULL values, so it will be null.
it should be Integer only if you allow NULL values